src/neoExpressHelper.ts

```diff
@@ -0,0 +1,209 @@
+                    resolve(StringResult.Error(error.message));
+                } else if (stderr) {
+                    console.error('Error creating instance', command, error, stderr);
+                    resolve(StringResult.Error(stderr));
+                } else {
+                    resolve(StringResult.Success(stdout));
+                }
+            });
+        });
+    }
+
+    public static async createWallet(
+        neoExpressJsonFullPath: string,
+        walletName: string,
+        allowOverwrite: boolean): Promise<StringResult> {
+
+        let command = shellEscape.default([
+            'neo-express',
+            'wallet',
+            'create', 
+            allowOverwrite ? '-f' : '']);
+        command += ' -i ' + NeoExpressHelper.doubleQuoteEscape(neoExpressJsonFullPath);
+        command += ' ' + NeoExpressHelper.doubleQuoteEscape(walletName);
+        return await new Promise((resolve) => {
+            childProcess.exec(command, (error, stdout, stderr) => {
+                if (error) {
+                    console.error('Error creating wallet', command, error, stderr);
+                    resolve(StringResult.Error(error.message));
+                } else if (stderr) {
+                    console.error('Error creating wallet', command, error, stderr);
+                    resolve(StringResult.Error(stderr));
+                } else {
+                    resolve(StringResult.Success(stdout));
+                }
+            });
+        });
+    }
+
+    public static async isNeoExpressInstalled(): Promise<boolean> {
+        let command = shellEscape.default([ 'neo-express', '-v' ]);
+        return await new Promise((resolve) => {
+            childProcess.exec(command, (error, stdout, stderr) => {
+                if (error) {
+                    resolve(false);
+                } else if (stderr) {
+                    resolve(false);
+                } else {
+                    resolve(true);
+                }
+            });
+        });
+    }
+
+    public static async showAccount(
+        neoExpressJsonFullPath: string,
+        wallet: string): Promise<JsonResult> {
+
+        let command = shellEscape.default(['neo-express', 'show', 'account']);
+        command += ' ' + NeoExpressHelper.doubleQuoteEscape(wallet);
+        command += ' -i ' + NeoExpressHelper.doubleQuoteEscape(neoExpressJsonFullPath);
+        command += ' -j';
+        return await new Promise((resolve) => {
+            childProcess.exec(command, (error, stdout, stderr) => {
+                if (error) {
+                    console.error('Error showing account', command, error, stderr);
+                    resolve(JsonResult.Error());
+                } else if (stderr) {
+                    console.error('Error showing account', command, error, stderr);
+                    resolve(JsonResult.Error());
+                } else {
+                    resolve(JsonResult.Success(stdout));
```

Comment:
does this json content get parsed elsewhere?

Reply:
it's parsed in the constructor for JsonResult